src/main/webapp/app/exam/participate/exercises/modeling/modeling-exam-submission.component.html

```diff
@@ -1,16 +1,22 @@
 @if (exercise) {
+    <h3 class="text-align-left fw-normal">
+        <span>
+            {{ exercise.title }}
+        </span>
+        <!-- prettier-ignore -->
+        <span left-header>&nbsp;({{ exercise.maxPoints }} {{ 'artemisApp.examParticipation.points' | artemisTranslate }}@if (exercise.bonusPoints) {
+            <span
+            >, {{ exercise.bonusPoints }} {{ 'artemisApp.examParticipation.bonus' | artemisTranslate }}</span>
+            }) @if (exercise.includedInOverallScore !== IncludedInOverallScore.INCLUDED_COMPLETELY) {
+            <jhi-included-in-score-badge
+            [includedInOverallScore]="exercise.includedInOverallScore" />
+          }</span>
+    </h3>
+    <hr />
```

Comment:
Due to the urgency of the PR this does not need to be changed now - the same change seems to occur for each exercise type, in such cases we should create a component and reuse it instead of duplicating the code

Reply:
I totally agree with you 👍